frontend/src/metabase/home/components/HomeContent/HomeContent.tsx

```diff
@@ -43,6 +52,10 @@ export const HomeContent = (): JSX.Element | null => {
   }
 
   if (embeddingHomepage === "visible" && user.is_superuser) {
+    if (isEE && isSimpleEmbeddingAvailable) {
```

Comment:
I don't think we need `isEE` here. If we have the token we can assume it's EE. I sampled some parts that use `useHasTokenFeature`, I don't see they have to check for `isEEBuild()`. This would make things a little cleaner and more precise. As this kind of imply OSS could have the token feature for some reason.

I also tested by not setting `MB_EDITION=ee` but leave the license there. Also run BE in OSS mode. `isSimpleEmbeddingAvailable` was `false` so I think we're safe to just check for the token feature alone.

Reply:
Ah yes we don't! If it's `"embedding"` then we do because of a backend bug that always return this token in OSS if a token is enabled, but if it's `"simple_embedding"` like in this case then we definitely don't. Let me fix this

Reply:
fixed in https://github.com/metabase/metabase/pull/63596/commits/7f854850eb04038eeea4036ff43e08b3815e8b09